hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/client/HdfsClientConfigKeys.java

```diff
@@ -414,6 +415,11 @@
 
     String  PREFETCH_SIZE_KEY = PREFIX + "prefetch.size";
 
+    String URI_CACHE_KEY = PREFIX + "uri.cache.enable";
+    boolean URI_CACHE_DEFAULT = false;
+    String URI_CACHE_EXPIRE_MS_KEY = PREFIX + "uri.cache.expire.ms";
```

Comment:
I think it's better not to have this config as I suggested in the JIRA. It's for two reasons:
1. This seems not critical parameter and Hadoop already has too many configurations. I think the default value 12hours is good enough. It's not meant to be tuned by per client since the cache is static and shared. So removing this config seems preferred to me.
2. The URI cache is initialized only once. So passing the `uriCacheExpireMs` parameter overtime does not really carry enough information. And it may be confusing if people mistakenly interpret this as a per-URI setting when they call `NetUtils.createSocketAddr()` which is not.

Thoughts?

Reply:
I'm so sorry, I misunderstood the meaning of jira. I thought the two parameters cache size and expire time seem a bit overkill, so the cache size was removed and the expire time was retained.
Now I will remove expire time and use 12 hours as the default value.

Reply:
Thanks, it's my bad. I could have made it clearer.

Reply:
Thanks for your review, I have fixed all the comments.